src/main/java/Duke.java

```diff
@@ -1,10 +1,94 @@
+import java.util.Scanner;
+import java.util.ArrayList;
 public class Duke {
+
+    //Definition lists //
+    public static final String margin = "*----------------------------*";
+    public static String updateStr = "added: ";          // for adding new tasks in taskList
+
+    public static Task[] taskList = new Task[100];  // Create array to store the tasks
+    public static int taskCount = 0;                    // Counter to track how many tasks in taskList
+
+
+    // Run program here: //
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+//        String logo = " ____        _        \n"
+//                + "|  _ \\ _   _| | _____ \n"
+//                + "| | | | | | | |/ / _ \\\n"
+//                + "| |_| | |_| |   <  __/\n"
+//                + "|____/ \\__,_|_|\\_\\___|\n";
+//        System.out.println("Hello from\n" + logo);
+
+
+        // Welcome Message //
+        System.out.println(margin);
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+        System.out.println(margin);
+
+        Scanner input = new Scanner(System.in);
+        String userCommand;
+
+        do{
+            userCommand = input.nextLine();
+
+            enter(userCommand);
+        }
+        while(!userCommand.equals("bye"));
+
+        }
+
+    // How the program runs based on user's input commands //
+    public static void enter(String userCommand) {
+
+        String[] strNumSeparator = userCommand.split(" ");  // split string & number components
+
+        switch (strNumSeparator[0]) {
+            case "bye":        //Exit the program
+                System.out.println(margin);
+                System.out.println("Bye. Hope to see you again soon!");
+                System.out.println(margin);
+                break;
+
+            case "list":       // View taskList
+                System.out.println(margin);
+                System.out.println("Here are the tasks in your list:");
+                for (int i = 0; i < taskCount; i++) {
```

Comment:
could use taskNumber instead of "i" to be clearer (for better readability)

Reply:
I think the coding standards mentioned how single letter iterators are allowed for loops. So I think this is okay :)